public/index.js

```diff
@@ -1,23 +1,28 @@
+'use strict';
+
 const $urlInput = $('.url-input');
 const $displayUrls = $('.display-urls');
```

Comment:
this is fixed 👍  I deleted the one I'd created, I missed that there was already one there.